compiler/test/compilable/ctod.i

```diff
@@ -15,7 +15,6 @@ extern (C)
 		SQL_IS_YEAR = 1,
 		SQL_IS_MONTH = 2,
 	}
-	alias SQLINTERVAL = enum SQLINTERVAL;
```

Comment:
This should be `alias SQLINTERVAL = int;`.

Reply:
No, I don't think so. That alias is for C code so the two SQLINTERVAL's can reside in two different symbol tables.

Reply:
This diff should be as it is, otherwise it results in 
```
onlineapp.d(9): Error: alias `onlineapp.SQLINTERVAL` conflicts with enum `onlineapp.SQLINTERVAL` at onlineapp.d(4)`
```

Reply:
That enum should be anonymous too.

https://github.com/dlang/dmd/pull/15731#discussion_r1372749847

Reply:
The enum should not be anonymous as it is a valid named type in both C and Di. The members should be aliased into the module scope though.